docs/deploy-to-vercel.md

```diff
@@ -1,27 +1,13 @@
 # Deploy to Vercel
 
-[Vercel](https://vercel.com) (formerly ZEIT) is a cloud platform for websites and serverless APIs, that you can use to deploy your Gridsome projects to your personal domain (or a free `.now.sh` suffixed URL).
+[Vercel](https://vercel.com) is a cloud platform that enables developers to host Jamstack websites and web services that deploy instantly, scale automatically, and requires no supervision, all with zero configuration. They provide a global edge network, SSL encryption, asset compression, cache invalidation, and more.
```

Comment:
"requires" -> "require"

Reply:
I believe `requires` is in parallel with `enables`.

Reply:
"require" is the verb, so it should be a singular or plural depending on if its subject is singular or plural. My understanding is that the subject is "websites and web services."

In other words:

>Vercel is a cloud platform that enables developers to host ... websites and web services that ... require no supervision.

It's the websites that require no supervision, not the cloud platform, right?

Reply:
Fixing!